steering_vectors/train_steering_vector.py

```diff
@@ -56,18 +60,39 @@ def train_steering_vector(
     layer_config = guess_and_enhance_layer_config(model, layer_config, layer_type)
     pos_activations: dict[int, list[Tensor]] = defaultdict(list)
     neg_activations: dict[int, list[Tensor]] = defaultdict(list)
+    if len(training_samples[0]) == 2:
+        training_samples = [
+            SteeringVectorTrainingSample(sample[0], sample[1], None, None)
+            for sample in training_samples
+        ]
+
+    def get_token_index(
```

Comment:
nit: This internal function doesn't reference anything in the outer scope, so I'd recommend moving this into a top-level function called `_get_token_index()`. This will clean up the `train_steering_vector()` function, make it clearer that this function is a simple pure function of its inputs, and make it possible to test this function explicitly in unit tests.

Reply:
Resolved by [50aed1b](https://github.com/steering-vectors/steering-vectors/pull/21/commits/50aed1b17e6b6beb9081b78b72d3373bd3465fa2)